lib/stage-autocompleter.js

```diff
@@ -121,9 +121,42 @@ class StageAutoCompleter {
    *
    * @returns {Function} The completion function.
    */
-  getCompletions(editor, session, position, prefix, done) {
+  async getCompletions(editor, session, position, prefix, done) {
     // Empty prefixes do not return results.
     if (prefix === '') return done(null, []);
+
+    // Comments block do not return results.
+    let value = session.getValue();
+    const findCommentsRe = new RegExp([
+      /\/(\*)[^*]*\*+(?:[^*\/][^*]*\*+)*\//.source, // multi-line comment
+      /\/(\/)[^\n]*$/.source, // single-line comment
+      /"(?:[^"\\]*|\\[\S\s])*"|'(?:[^'\\]*|\\[\S\s])*'/.source, // string
+      /(?:[$\w\)\]]|\+\+|--)\s*\/(?![*\/])/.source, // division operator
+      /\/(?=[^*\/])[^[/\\]*(?:(?:\[(?:\\.|[^\]\\]*)*\]|\\.)[^[/\\]*)*?\/[gim]*/.source
+    ].join('|'), // regex
+    'gm' // global and multiline flags
+    );
+
+    const activePosition = session.getDocument().positionToIndex(position, 0);
+    let isComment = false;
+    const findCommnets = () => new Promise((resolve) => {
```

Comment:
Promises are also not supported in IE. Also typo here, should be `findComments`. But let's stick with standard callbacks for the autocomplete.